### DISCUSSION TITLE
Update Markor to latest version, what is the best place?

### DISCUSSION THREAD

**Discussion by aisbergde** [gsantner/markor#1564] [Q&A] [answered]
In the Markor repository I see new versions from time to time.  
In Google Playstore the latest version is 2.8.0
In the Readme is also a link to f-droid, and on the Markor website https://f-droid.org/packages/net.gsantner.markor/ the latest version is 2.8.5

2.8.0 is not available at all.

I wanted to not directly download the APK because of the hint, that I should better use the f-droid app to get updates. So I installed it. But in this app I get the information, that my Markor version is up to date and the last available version is 2.8.0.

Now I am confused, why the information in the f-droid is different from the information on the website. Or maybe these are 2 different versions, on official and one for beta testing?

What is the recommended way to use which version of Markor?

**Comment by gsantner** [comment, marked as answer]
I don't update Google Play anymore. [F-Droid](https://f-droid.org/packages/net.gsantner.markor/) or [GitHub](https://github.com/gsantner/markor/releases).  
There is only one line of releases, there is no beta track.  
Generally, if you installed from Google Play you can download and upgrade from [GitHub](https://github.com/gsantner/markor/releases). (GitHub binaries are signed by me, using the same way&cert at gplay).

**Reply by kenclark75** [reply]
I am an enthusiastic user of Markor so I was very disappointed when I read that you would no longer be updating the app on the Google Play Store.
However, I recently received an automatic update to v2.10.1 (136).
Could you please clarify whether you will continue to support the app though the Play Store.

**Comment by aisbergde** [comment]
OK, then I still don't understand, why the latest version is only shown on the. f-droid website, but not in their app. But it looks like this could be a f-droid issue. Or do you marked 2.8.0 on f-droid as release, and other versions as pre-release and that's why the latest versions are not shown in the app?

If now f-droid is the recommended way: Could you check in your f-droid client, which version is shown there?

**Reply by gsantner** [reply]
The latest release is available there, you might need to uninstall first to switch to f-droid signing key based installation.  

As said, latest release is available there (usually after 3 days of release), everything else is not up to Markor.

**Comment by aisbergde** [comment]
I tried on an old phone, f-droid client shows 2.3.3 as "current version" and no new version. It looks like the f-droid just detects the last version, and if it was installed from PlayStore then f-droid doesn't suggest to upgrade.

I uninstalled it. Now f-droid shows no version but 
"Keine Version mit kompatibler Signatrur"
and a bit later I see the "2.2.7" as "vorgeschlagen". And if I try to install using the f-droid app, the apk is not available (of course, because 2.2.7 is a very old version).

it looks like the f-droid is a very buggy solution to get the latest version of software. A user gets informed, "all apps are up to date", but the reality is quite different and on the web there are newer versions. After some restarts I finally got the latest version via f-droid. But only because I know that there is a newer version and I uninstalled Markor before.

I never had these kinds of issues with Google Playstore. Why did you decide to not update the App on Google Playstore? Are there issues with Google playstore?

**Reply by gsantner** [reply]
Still my stance stays, the main installation source of Markor is  [F-Droid](https://f-droid.org/en/packages/net.gsantner.markor/). Everything else might or might not get new versions.

**Comment by aisbergde** [comment]
I lost all my settings, and forgot the backup of settings before uninstalling ... :-(

**Reply by gsantner** [reply]
the good thing is - settings are easy changeable. data isn't gone.

**Comment by aisbergde** [comment]
Finally, I got the latest version. line break in folder lists works perfectly! I am incredibly happy!
